DID_CCF.md

```diff
@@ -122,10 +124,11 @@ De-registering a domain does _not_ invalidate DIDs previously created in the dom
 ### Create (Private API)
 All authenticated members of a consortium and registered users of a given CCF network can create new decentralized identifiers by calling the **identifiers/create** endpoint. This API generates a cryptographic key pair and DID Document which is stored in the members confidential computing enclave.
 
-```
-./scurl.sh https://<host>/app/identifiers/create --cacert <service_certificate>.pem --signing-key <member_private_key.pem> --signing-cert <member_certificate.pem> -H "content-type: application/json" -X POST
+The endpoints would be used as follows where the data-binary is the buffer/byte array of the signed cose of an empty payload. Alternativly JWTs could be used, for examples see [JWT Authentication CCF Documentation](https://microsoft.github.io/CCF/main/build_apps/auth/jwt.html).
+```sh
+curl https://<host>/app/identifiers/create --cacert <service_certificate>.pem --data-binary @- -H "content-type: application/cose" -X POST

```

Comment:
Does `ccf_cose_sign1` need to be called before this?

Reply:
ccf_cose_sign1 specifically handles governance specific requests, so the governance etc. 
This is because it expects specific headers related to this, e.g a governance request could start: 
- ccf_cose_sign1 --ccf-gov-msg-type ack --ccf-gov-msg-created_at `date -Is`. 
ccf-gov-msg-type etc are specific to governance requests and are required by ccf_cose_sign1. They are not a requirement of standard COSE signing as used by the DID:CCF app. Which is why I have excluded it from here as it is up to a user what they use to create the cose signed request.

Reply:
I think it'd be good to provide an example of how a user could create a COSE-signed request, especially if it can be done from the command-line (?); it's entirely possibly that many of those who arrive here will be unfamiliar with it.